app/views/pages/developer-guide/pos-marketplace-template/tests.liquid

```diff
@@ -94,9 +95,83 @@ Then open your browser and you can visit it at:
 Put tests into `app/views/partials/test/commands`, files should end with `_test.liquid`.
 
 To run all tests, deploy code on Instance and go to page:
-`https://your-instance.example.com/tests/run`
-    
+`https://your-instance.example.com/tests`
 To run one test go to page:
 `https://your-instance.example.com/tests/run?name=commands/items/create/check_test`
-    
-Use assertion partials from `app/views/partials/test/assertions`.
\ No newline at end of file
+
+Use assertion partials from `app/views/partials/test/assertions`.
+
+**Example of unit test file with user creating/deleting:**
+
+In this example, you will assign json data to two variables. The first one is for the user data, and the second one is for the profile data.
+
+User
+```js
+  % parse_json payload %
+  {
+    "username": "user name",
+    "email": "test@example.com",
+    "password": {{ "password" | json }},
+    "agreed": "true"
+  }
+  % endparse_json %
+```
+
+Profile
+```js
+  % parse_json profile %
+  {
+    "first_name": "first name",
+    "last_name": "last name",
+    "phone": "1234234"
+  }
+  % endparse_json %
+```
+
+Then you create the user along with this function:
+
+`#21 function user = 'lib/commands/users/create', object: payload, profile: profile`
+
+Validation check:
+
+```js
+unless user.valid
+  include 'test/register_error', contract: contract, field_name: 'user#valid', message: user.errors
+endunless
+```
+
+The test will try to find the user in the database by user ID:
+
+```js
+function u = 'lib/queries/users/find', id: user.id
+
+unless u
+  include 'test/register_error', contract: contract, field_name: 'primary user', key: 'app.test.should.be_valid'
+endunless
+```
+
+The test will try to create the user with the same credentials. It checks whether user2 is created.
+
+```js
+function user2 = 'lib/commands/users/create', object: payload
+
+if user2.valid
+  include 'test/register_error', contract: contract, field_name: 'extra user', key: 'app.test.should.not.be_valid'
+endif
+
+if user2.id
+  include 'test/register_error', contract: contract, field_name: 'extra user', key: 'app.test.should.not.be_valid'
+endif
+```
+
+The test deletes the user, and checks if the user exists in the database by user ID.
+
+```js
+function deleted = 'lib/commands/users/delete', object: u
+
+function u = 'lib/queries/users/find', id: user.id
+
+if u
+  include 'test/register_error', contract: contract, field_name: 'deleted user', key: 'app.test.should.be_valid'
+endif
+```
```

Comment:
From experience, readers like to copy paste code, so it would be nice if the examples included at least parsable code, which means either {% %} or {% liquid ... %}